SPECS/mariner-rpm-macros/mariner-rpm-macros.signatures.json

```diff
@@ -12,7 +12,6 @@
   "gpgverify": "db0e050f56b694497d70603a6f5c17dd60ddbcf7cee670616851cd389f6767c4",
   "macros": "b7ad5c17d6ce105427eeef54f43317cb1cc8cc114bfeca7693069242ad747d20",
   "macros.check": "79367176c3c7d10c0158b6e5d881e0fc3c8fd50c5957dad2f097c2d4a37833e7",
-  "macros.dist": "817653f151349adff8c658143cf01ad1f8b51168be2087e4e02778224da85d63",
```

Comment:
Does the full toolchain and packages build show no regressions after this change? I believe we should be fine but want to make sure.

Reply:
here's a fresh pipeline run:
https://dev.azure.com/mariner-org/mariner/_build/results?buildId=513845&view=results